cdap-ui/app/features/hydratorplusplus/services/hydrator-service.js

```diff
@@ -49,7 +49,13 @@ class HydratorPlusPlusHydratorService {
     });
     connections = config.connections;
     // Obtaining layout of graph with Dagre
-    var graph = this.DAGPlusPlusFactory.getGraphLayout(nodes, connections);
+    var graph;
+    if (isStudio) {
+      graph = this.DAGPlusPlusFactory.getGraphLayout(nodes, connections, 100);
```

Comment:
Is there any specific reason why we are not having this for both studio and detailed view ?